entity-types/infra-azurevpngateways/golden_metrics.yml

```diff
@@ -0,0 +1,36 @@
+averageBandwidth:
+  title: Gateway S2S Bandwidth
+  unit: BYTES_PER_SECOND
+  queries:
+    azure:
+      select: max(azure.network.vpngateways.AverageBandwidth)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
+tunnelAverageBandwidth:
+  title: Average bandwidth of a tunnel in bytes per second
+  unit: BYTES_PER_SECOND
+  queries:
+    azure:
+      select: max(azure.network.vpngateways.TunnelAverageBandwidth)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name     
+tunnelEgressBytes:
+  title: Outgoing bytes of a tunnel
+  unit: BYTES
+  queries:
+    azure:
+      select: max(azure.network.vpngateways.TunnelEgressBytes)
```

Comment:
I wonder why here `max` is used, whereas in the following metric `sum` is used instead. Is it intentional?